ccmlib/scylla_cluster.py

```diff
@@ -9,6 +9,7 @@
 import datetime
 
 from six import print_
+from packaging.version import Version
```

Comment:
this package isn't part of ccm dependency (you are adding it in dtest)

please check if you can use this:
```
from distutils.version import LooseVersion 
```